### PR TITLE
The FileUtils module adds exported interfaces and removes unexported …

### PR DESCRIPTION
**Pull request by qiuguohua**

### CODE REVIEW

File: @types/jsb.d.ts [hunk 1]
```diff
@@ -663,17 +597,35 @@ declare namespace jsb {
...
+        /**
+         *  Renames a file under the given directory.
          *
-         *  @param dirPath The path of the directory, it could be a relative or an absolute path.
-         *  @return File paths in a string vector
+         *  @param oldFullpath  The current fullpath of the file. Includes path and name.
+         *  @param newFullPath  The new fullpath of the file. Includes path and name.
+         *  @return True if the file have been renamed successfully, false if not.
          */
-        export function listFilesRecursively (dirPath:string, files:Array<string>):void;
```
Comment (PatriceJiang) [inline @types/jsb.d.ts]: This interface is exported in jsb_cocos_manual.cpp

Comment (qiuguohua) [reply @types/jsb.d.ts]: ok, i'll add

**Review by PatriceJiang — APPROVED** [review]

File: @types/jsb.d.ts [hunk 2]
```diff
@@ -673,7 +607,32 @@ declare namespace jsb {
...
+        export function getDataFromFile(fullpath: string):ArrayBuffer;
+
+        /**
+         * write Data into a file
+         *
+         *@param data the data want to save
+         *@param fullpath The full path to the file you want to save a string
+         *@return bool
+         */
+        export function writeDataToFile(buffer: ArrayBuffer, fullpath: string):ArrayBuffer;
```
Comment (PPpro) [inline @types/jsb.d.ts]: What should be the return type of this function ? bool or  ArrayBuffer ?

Comment (qiuguohua) [reply @types/jsb.d.ts]: It's bool,

**Review by minggo — APPROVED** [review]

**Review by PPpro — APPROVED** [review]

**Pull request merged**